app/Http/routes.php

```diff
@@ -80,5 +80,6 @@
  /*-----------------------*
   *  Settings Controller  *
   *-----------------------*/
+Route::get('settings', 'PagesController@settings');
```

Comment:
this should be `/settings` instead of `settings`. without the `/` it's a relative link